### PR TITLE
add command for accept next word in copilot

### PR DESCRIPTION
**Pull request by kevinushey**
Addresses https://github.com/rstudio/rstudio/issues/13522.

### CODE REVIEW

**Review by jgutman — COMMENTED** [review]
The build is now failing with 

```
src/gwt/test/org/rstudio/studio/client/workbench/commands/DummyCommands.java:6: 
error: DummyCommands is not abstract and does not override abstract method 
               copilotAcceptNextWord() in Commands
```

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetCopilotHelper.java [hunk 1]
```diff
@@ -295,7 +297,42 @@ else if (event.getKeyCode() == KeyCodes.KEY_BACKSPACE)
    @Handler
    public void onCopilotRequestCompletions()
    {
-      completionTimer_.schedule(0);
+      if (display_.isFocused())
```
Comment (jgutman) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetCopilotHelper.java:300]: good catch

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java [hunk 2]
```diff
@@ -1192,14 +1192,15 @@ public void scrollToY(int y, int animateMs)
       fireEvent(new ScrollYEvent(Position.create(getFirstVisibleRow(), 0)));
    }
 
-   public void insertCode(String code)
+   public void insertCode(String code, boolean unused)
```
Comment (jgutman) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java:1195]: We don't have like a naming convention for this in Java, like `_unused` ?

**Review by jgutman — APPROVED** [review]
LGTM, pending fixing the issue causing the build failure. Does this just create the command, or does this also create the way to trigger it via keyboard shortcut ? (it doesn't seem like it does the latter, but maybe I'm missing something) We should add to NEWS (at least once this can be triggered by the user)

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java (hunk 2 above)
Comment (kevinushey) [reply src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditor.java:1195]: As far as I know, we don't, at least for the Java sources.

**Pull request merged**